src/R/Components/Test/Plots/PlotIntegrationTest.cs

```diff
@@ -223,26 +223,6 @@ public class PlotIntegrationTest : IAsyncLifetime {
 
         [Test(ThreadType.UI)]
         [Category.Plots]
-        public async Task ExportAsPdfInvalidFilename() {
```

Comment:
This test was written to verify file paths on the R side. This is no longer necessary. The filename used for export here must satisfy windows path requirements. The save dialog box disallows any invalid path characters. On the R side we use tempfile() to generate a temporary file name. Discussed removal of this test with @huguesv.